crates/viewer/re_space_view_dataframe/src/time_range_table.rs

```diff
@@ -87,23 +86,25 @@ pub(crate) fn time_range_table_ui(
     }
 
     #[inline]
-    fn entity_components_to_key_value_iter<'a>(
+    fn entity_to_key_value_iter<'a>(
         ctx: &ViewerContext<'_>,
         entity_path: &'a EntityPath,
-        component: &'a ComponentName,
         timeline: Timeline,
         resolved_time_range: ResolvedTimeRange,
     ) -> impl Iterator<Item = (RowKey, Arc<Chunk>)> + 'a {
         let range_query = RangeQuery::new(timeline, resolved_time_range);
 
         ctx.recording_store()
-            .range_relevant_chunks(&range_query, entity_path, *component)
+            .range_relevant_chunks_for_all_components(&range_query, entity_path)
             .into_iter()
             // Exploit the fact that the returned iterator (if any) is *not* bound to the lifetime
             // of the chunk (it has an internal Arc).
             .map(move |chunk| {
-                let chunk = Arc::new(chunk.range(&range_query, *component));
-                (Arc::clone(&chunk).iter_indices_owned(&timeline), chunk)
+                let all_indices = Arc::clone(&chunk)
+                    .iter_indices_owned(&timeline)
+                    .filter(move |(time, _)| range_query.range.contains(*time));
```

Comment:
@teh-cmc here I wanted to use `Chunk::range()`, but this one takes a `ComponentName`, which doesn't really makes sense now (is `range` forcing a single-component PoV here?). That's why I did the filtering "by hand".

Reply:
nvm, I was already doing that filtering manually.